pkgs/misc/translatelocally-models/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  modelSpecs = (builtins.fromJSON (builtins.readFile ./models.json));
+  withCodeAsKey = f: { code, ... }@attrs: lib.nameValuePair code (f attrs);
+  mkModelPackage = { name, code, version, url, checksum }:
+    stdenv.mkDerivation {
+      pname = "translatelocally-model-${code}";
+      version = toString version;
+
+      src = fetchurl {
+        inherit url;
+        sha256 = checksum;
+      };
+      dontUnpack = true;
+
+      installPhase = ''
+        TARGET="$out/share/translateLocally/models"
+        mkdir -p "$TARGET"
+        tar -xzf "$src" -C "$TARGET"
```

There is a shell file in there which won't work, and it's not even consistently marked executable in all languages. This avoids patching its shebang and in turn allow the official nix cache to deduplicate/hardlink the outputs which gets rebuilt everytime `stdenvNoCC` is updated more effectively.

```suggestion
        tar -xzf "$src" -C "$TARGET"
        find "$out" -type f -exec chmod -x {} +
```